bin/reth/src/node/events.rs

```diff
@@ -138,18 +138,22 @@ impl NodeState {
     }
 
     fn handle_consensus_layer_health_event(&self, event: ConsensusLayerHealthEvent) {
-        match event {
-            ConsensusLayerHealthEvent::NeverSeen => {
-                warn!("Post-merge network, but never seen beacon client. Please launch one to follow the chain!")
-            }
-            ConsensusLayerHealthEvent::HasNotBeenSeenForAWhile(period) => {
-                warn!(?period, "Post-merge network, but no beacon client seen for a while. Please launch one to follow the chain!")
-            }
-            ConsensusLayerHealthEvent::NeverReceivedUpdates => {
-                warn!("Beacon client online, but never received consensus updates. Please ensure your beacon client is operational to follow the chain!")
-            }
-            ConsensusLayerHealthEvent::HaveNotReceivedUpdatesForAWhile(period) => {
-                warn!(?period, "Beacon client online, but no consensus updates received for a while. Please fix your beacon client to follow the chain!")
+        // If pipeline is running, consensus layer messages are expected to be neither received nor
```

Comment:
this is not correct, they're always handled.

Reply:
right, removed the part about processing. We always process incoming messages, but not expect to receive them when the pipeline is running.

Reply:
I also don't think this is correct, we're still getting FCUs, at least with lighthouse, so perhaps this is a client specific thing.

the change makes sense, but comment should reflect this

Reply:
hmm, maybe "If pipeline is running, it's fine to not receive any consensus layer messages."?